README.md

```diff
@@ -3,19 +3,38 @@ Goal: To bootstrap a light-weight Jenkins instance with auto-scaling replicas us
 
 *NOTE: This is a work in progress! The README will be updated with functional steps as they are added.*
 
-## Create AWS infrastructure
-Running this setup script will create a security group in your chosen VPC, an EC2 instance,
-and a DynamoDB table and s3 bucket for remote terraform state management.
+## Replace custom variables for your AWS account
+Update the following values in `terraform/backend.auto.tfvars`: 
+* `bucket` - Where your Terraform state will be stored. Must be globally unique!
 
-You will be prompted by Terraform to confirm you want to apply the AWS resources, and then to move Terraform state from
-local to remote s3 backend.
+Update the following values in `terraform/network.auto.tfvars`:
+* `vpc_id` - VPC that Jenkins will run in.
+* `subnet_id` - ID of a public subnet in your VPC.
+* `ingress_cidr` - CIDR address for inbound traffic to your Jenkins instance.
```

Comment:
Love some example values to show syntax for all the values.   i.e 192.168.0.0/24